src/agent/src/image.rs

```diff
@@ -66,26 +75,41 @@ impl ImageService {
         if !guest_pause_bundle.exists() {
             bail!("Pause image not present in rootfs");
         }
+        let guest_pause_config = guest_pause_bundle.join(CONFIG_JSON);
```

Comment:
Hi @ChengyuZhu6 ! Not a Rust expert speaking but... I noticed that some path operations in `image.rs` have used [safe_path](https://docs.rs/safe-path/latest/safe_path/index.html)::[scoped_join](https://docs.rs/safe-path/latest/safe_path/fn.scoped_join.html#) instead of `join`. Perhaps we should use it in this function as well to make it safer and consistent?

Reply:
Done.